scripts/netlify/changes-in-docs-since-master

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Compare current branch with master to see if there are changes in the docs/
+# folder
+set -x
+
+# Remote urls are not always set on Netlify
+git remote set-url origin https://github.com/algolia/docsearch.git
+git fetch
+
+CHANGES_IN_DOCS="$(git diff --name-only origin/master | grep '^docs/')"
+if [ "$CHANGES_IN_DOCS" = "" ]; then
```

Comment:
```suggestion
if [ $CHANGES_IN_DOCS = "" ]; then
```

Reply:
Not tested but I think it should be set this way

Reply:
If you don't set the quotes, it will fail if for some reason `$CHANGES_IN_DOCS` contains spaces or glob characters (like `*`). In that case it _shouldn't_ happen, but it's best practice in bash to always quote strings you want to test.

I actually use [shellcheck](https://www.shellcheck.net/) in my IDE to warn me about those issues directly

Reply:
Thank you for the explanation. I have noticed it and removed this review.
Thanks for the TIP